.github/workflows/_buildpacks-release.yml

```diff
@@ -279,7 +331,7 @@ jobs:
           token: ${{ secrets.cnb_registry_token }}
           id: ${{ matrix.buildpack_id }}
           version: ${{ matrix.buildpack_version }}
-          address: ${{ matrix.docker_repository }}@${{ steps.digest.outputs.value }}
+          address: ${{ matrix.image_repository }}@${{ steps.digest.outputs.value }}
 
   update-builder:
```

Comment:
Could we provide a [job summary](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary) here that links to the builder PR that gets created?

Reply:
Good idea. But I think I'll decline that for this PR. I'm not making significant changes to this step and this PR is already massive. I did add job summaries for matrix generation, temporary tags, and stable tags.